testsuite/config_reader.py

```diff
@@ -1,13 +1,16 @@
 # pylint: disable=missing-docstring
 import configparser
-from flask import request
+from flask import request, session
 
 
-def get_config():
+def get_config(default='smart'):
     config = configparser.ConfigParser()
 
     try:
-        config.read_string(request.form['config'])
+        vendor = request.form.get('vendor', default).lower()
```

Comment:
Yeah, it's really unfortunate, but the way behave actually invokes the test suite makes it really difficult to be explicit about dependencies like that. If I remember right, they're basically calling `eval` on environment.py. It makes sense on one hand, since it makes it impossible to accidentally share state between tests, but it would be really useful if I could pass environments into the test suite :/